ote/src/clj/ote/services/transport.clj

```diff
@@ -164,13 +174,16 @@
                                ::t-operator/type            "organization"
                                ::t-operator/approval_status "approved"
                                ::t-operator/is_organization true})
-        member (specql/insert! db ::user/member
-                               {::user/id         (str (UUID/randomUUID))
-                                ::user/table_id   (get-in user [:user :id])
-                                ::user/group_id   (:ote.db.transport-operator/group-id group)
-                                ::user/table_name "user"
-                                ::user/capacity   "admin"
-                                ::user/state      "active"})]
+        member (create-member! db user-id group)
+        ;; Ensure that other users get permissions to new operator as well
+        ;; Get other possible users that need permissions
+        other-users (fetch-users-within-same-business-id-family db {:business-id (::t-operator/business-id op)
+                                                           :user-id user-id})]
+    (when other-users
```

Comment:
Would following (or similar if this doesn't work) eliminate the need for explicit `when` and `doall`?
```(for [usr other-users]
    (create-member! db (:user-id usr) group))```